platform/linux-generic/include/odp_queue_internal.h

```diff
@@ -77,19 +63,18 @@ struct queue_entry_s {
 	char              name[ODP_QUEUE_NAME_LEN];
 };
 
-union queue_entry_u {
+typedef union queue_entry_u {
 	struct queue_entry_s s;
 	uint8_t pad[ROUNDUP_CACHE_LINE(sizeof(struct queue_entry_s))];
-};
-
+} queue_entry_t;
 
 queue_entry_t *get_qentry(uint32_t queue_id);
 
-int queue_enq(queue_entry_t *queue, odp_buffer_hdr_t *buf_hdr);
-odp_buffer_hdr_t *queue_deq(queue_entry_t *queue);
+int _queue_enq(queue_t handle, odp_buffer_hdr_t *buf_hdr);
+odp_buffer_hdr_t *_queue_deq(queue_t handle);
```

Comment:
There is a need. As I said earlier, before this patch, there were 2 sets of functions, for ex: **odp_queue_enq** and **queue_enq**. The **odp_** prefix needs to be removed for this patch. Hence the function names are changed, for ex: **queue_enq** and **_queue_enq**.

Now, **_queue_enq** is an API towards internal components. But it is called indirectly via the function pointer table, hence there is no need to add the prefix **__odp**, just **_** prefix is enough.